syntax = "proto3";

service Greeter {
    // Sends a greeting
    rpc SayHello (HelloRequest) returns (HelloReply) {}

    // Sends another greeting
    rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    string name = 1;
    int32  qw = 2;
}

message HelloReply {
    string message = 1;
}